docs/admin/secrets-setup.rst

```diff
@@ -106,29 +108,19 @@ Static secret sources
 =====================
 
 Generated secrets can, as the name suggests, be generated by Phalanx tooling during installation.
-Static secrets cannot; the Phalanx administrator has to provide the values of static secrets in order to install a Phalanx environment.
-
-Using the old installer
------------------------
-
-Currently, the installer scripts that populate Vault for a given Phalanx environment support either retrieving static secrets from 1Password_ or prompting interactively for them.
-For all SQuaRE-run Phalanx environments, the static secrets are stored in 1Password.
-
-This mechanism is being retired and therefore is not documented here in detail.
-For instructions on how to sync secrets using the installer scripts and using 1Password as a source of static secrets, see :ref:`dev-add-onepassword`, step three.
-
-Using the Phalanx command-line tool
------------------------------------
+Static secrets cannot be; the Phalanx administrator has to provide the values of static secrets in order to install a Phalanx environment.
```

Comment:
The first sentence combined with the "Static secrets cannot be" is maybe a bit awkward. Consider defining what a static secret is, and then make the comparison to generated secrets. For folks like me who don't make much, if any, use of secrets generation, defining static secrets via comparison isn't as effective.